site/content/en/guides/ecosystem/oam/_index.md

```diff
@@ -0,0 +1,211 @@
+
+#### Create setter by App Developer
+
+Let's say the developer need to claim two fields as "configurable" for his application, he can add two kpt setters here:
+
+```shell
+$ kpt cfg create-setter sampleapp/ instance-name example-component --field "metadata.name" --description "use to set an instance name" --set-by "sampleapp developer"
+```
+
+```shell
+$ kpt cfg create-setter sampleapp/ image nginx:1.16.1 --field "image" --description "use to set image for component" --set-by "sampleapp developer"
+```
+
+Then the app operator could see which parameters are available in this component like below:
+
+```shell
+$ kpt cfg list-setters sampleapp/
+  NAME              VALUE               SET BY                   DESCRIPTION             COUNT
+  image           nginx:1.16.1        sampleapp developer   use to set image for component   1
+  instance-name   example-component   sampleapp developer   use to set an instance name      1
+```
+
+It's very clear and easy to understand.
+
+#### Set Value by App Operator
+
+Then the application operator could set `instance-name` with a new name like this:
+
+```shell
+$ kpt cfg set sampleapp/ instance-name test-component
+set 1 fields
+```
+
+Check the component and you will find the instane name has been changed.
+
+```shell
+$ cat sampleapp/component.yaml
+apiVersion: core.oam.dev/v1alpha2
+kind: Component
+metadata:
+  name: test-component # {"$ref":"#/definitions/io.k8s.cli.setters.instance-name"}
+spec:
+  parameters:
+  - name: instance-name
+    fieldPaths:
+    - metadata.name
+...
+```
+
+### App Overview
+
+With kpt, you could see an overview of your App.
+
+```shell
+$ kpt cfg count sampleapp
+ApplicationConfiguration: 1
+Component: 1
+```
+
+So in the sampleapp, we have one ApplicationConfiguration and Component.
+
+### Live apply
+
+kpt includes the next-generation **apply** commands developed out of the Kubernetes [cli-utils](https://github.com/kubernetes-sigs/cli-utils) repository as the [`kpt live apply`](https://googlecontainertools.github.io/kpt/reference/live/apply) command.
+
+This means with `kpt live apply` command, we could wait for the controller reconcile.
+
+```shell
+$ kpt live init sampleapp
+Initialized: ../sampleapp/grouping-object-template.yaml
+```
+
+```shell
+$ kpt live apply sampleapp --wait-for-reconcile
+configmap/grouping-object-a4e18a93 created
+applicationconfiguration.core.oam.dev/example-appconfig configured
+component.core.oam.dev/test-component created
+3 resource(s) applied. 2 created, 0 unchanged, 1 configured
+configmap/grouping-object-a4e18a93 is Current: Resource is always ready
+applicationconfiguration.core.oam.dev/example-appconfig is Current: Resource is current
+resources failed to the reached Current status
+configmap/grouping-object-a4e18a93 pruned
+2 resource(s) pruned
+```
+
+Currently `kpt live apply` seems not work well and will be fixed soon by [issue #498](https://github.com/GoogleContainerTools/kpt/issues/498).
```

Comment:
The issue seems to be resolved.  Can you check if has been fixed in the latest release, then we could remove this line?

Reply:
yeah, I tried and removed that line. Thanks!